internal/component/azure/servicebus/subscription.go

```diff
@@ -193,6 +247,87 @@ func (s *Subscription) ReceiveAndBlock(handler HandlerFunc, lockRenewalInSec int
 	}
 }
 
+// ReceiveMultipleAndBlock is a blocking call to receive multiple messages on an Azure Service Bus subscription from a topic or queue.
+func (s *Subscription) ReceiveMultipleAndBlock(handler BulkHandlerFunc, lockRenewalInSec int, onFirstSuccess func()) error {
```

Comment:
This method is very similar to `ReceiveAndBlock` - can they be implemented in a way that share most of the code?

Reply:
Done!